src/React/Resources/assets/src/render_controller.ts

```diff
@@ -14,7 +14,7 @@ import { createRoot } from 'react-dom/client';
 import { Controller } from '@hotwired/stimulus';
 
 export default class extends Controller {
-    readonly componentValue: string;
+    readonly componentValue: string | undefined;
```

Comment:
```suggestion
    readonly componentValue?: string;
```

To be consistent with `propsValue`. I believe these are identical.

Reply:
I believe too.